packages/main/src/TableRow.ts

```diff
@@ -89,6 +98,11 @@ class TableRow extends TableRowBase {
 		} else {
 			this.removeAttribute("aria-current");
 		}
+		if (this.movable) {
+			this.setAttribute("draggable", "true");
+		} else {
+			this.removeAttribute("draggable");
+		}
```

Comment:
Better do this in TableRow.hbs, to encapsulate this attribute in a shadow DOM.

Reply:
Because of the structure of our Table components, roles and styles are applied to the Light DOM (to ensure better ACC and less shadow DOM nesting). Therefore we need to set draggable on the light DOM element